server/src/api_endpoints/clients/index.js

```diff
@@ -0,0 +1,70 @@
+const createError = require('http-errors')
+const dbAdminRole = require('database/layer/admin-client')
+
+/**
+ * Client Admin function to create a Client
+ *
+ * @param {*} req
+ * @param {*} res
+ * @param {*} next
+ */
+ const createClient = async (req, res, next) => {
+  const tenantName = req.body.tenant_name
+  const clientId = req.body.client_id
+  const name = req.body.name
+  const description = req.body.description
+  const secret = req.body.secret
+
+  try {
+    const client = await dbAdminRole.insertClient(tenantName, clientId, name, description, secret)
+    // change error handling to thrown exception
+    if (!client?.client_id) {
+      throw new Error(`Error creating client: ${client}`)
+    }
+
+    res.status(201).send({
+      client_id: client.client_id,
+      clientname: client.name,
+      clientdescription: client.description || '',
+      secret: client.secret
+    })
```

Comment:
I'm not a fan of using `clientname` and `clientdescription`. It seems unnecessarily verbose in naming
```suggestion
    res.status(201).send(client)
```

Reply:
@k1ch I updated the field `description`, as that is what it's supposed to be. 
We are currently using `clientname` in our datalake-api code so this would be a breaking change, unless we want to break consistency and have the new APIs return `name` and the existing `/self/clients` have a different response. 

Otherwise that would probably be a change by itself.

Reply:
Correct, when we update `/self/clients` to be consistent and use `name` instead of `clientname` that would be a breaking change. However, for this new API, I believe we can use `name` without any issue.

Reply:
@k1ch ok made the change, will commit shortly. The `/self/clients` actually only returns the client id and name so I think its fair to consider that one API a one-off.